esinstall/src/index.ts

```diff
@@ -363,12 +363,10 @@ ${colors.dim(
     treeshake: {moduleSideEffects: 'no-external'},
     plugins: [
       rollupPluginAlias({
-        entries: Object.entries(installAlias)
-          .filter(([, val]) => isPackageAliasEntry(val))
-          .map(([key, val]) => ({
-            find: key,
-            replacement: val,
-          })),
+        entries: Object.entries(installAlias).map(([key, val]) => ({
```

Comment:
+1 on this change, conceptually. But then we'll need to do this filtering near the `install()` call inside of Snowpack.

```diff
  // snowpack/src/commands/install.ts
- alias: config.alias, 
+ alias: filterPackageAliases(config.alias),  
```

Reply:
I'm confused. If we still do the filtering, just in a different place, then that brings us back to square one, with no actual behavior changed.

Can you help me understand why this filtering is being done? I don't understand why we'd want to make only _some_ aliases apply within dependencies and not others, and removing this filtering does seems to fix that issue with no adverse side effects.